source/aperture.f90

```diff
@@ -1021,20 +1006,20 @@ subroutine lostpart(turn, i, ix, llost, nthinerr)
           lparID = .false.
           jjx=1
 
-#ifdef COLLIMAT
-          do jj=1,npart
```

Comment:
@amereghe This bit

Reply:
https://github.com/SixTrack/SixTrack/blame/master/source/aperture.f90

Reply:
I'll let you sort this out in a different PR :)